File: Year 2/Semester 2/MPP/C#Socket/Model/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Model
{
    [Serializable]
    public class Ticket : Entity<int>
    {
        private string client_name;
        private string address;
        private int flightId;

        public Ticket(string client_name, string address, int flight)
        {
            this.client_name = client_name;
            this.address = address;
            this.flightId = flight;
        }
        public string Client_name { get { return client_name; } }
        public string Address { get { return address; } }
        public int FlightId { get { return flightId; } }
        
        public override string ToString()
        {
            return "Ticket: " + client_name + " " + address + " " + flightId;
        }
    }
}

File: Year 2/Semester 2/MPP/Protobuf/ProtoJava/AirlineModel/src/main/java/airline/model/Ticket.java
package airline.model;

import java.io.Serializable;

public class Ticket extends Entity<Integer> implements Serializable{
    private String client_name;
    private final String address;
    private final int flightId;

    public Ticket(String client_name,String address, int flightId) {
        this.client_name = client_name;
        this.address = address;
        this.flightId=flightId;
    }

    public String getClient_name() {
        return client_name;
    }

    public String getAddress() {
        return address;
    }

    public int getFlightId(){
        return flightId;
    }

    @Override
    public String toString() {
        return "Ticket{" +
                "client_name='" + client_name + '\'' +
                ", address='" + address + '\'' +
                ", flightId=" + flightId +
                '}';
    }

    @Override
    public int hashCode() {
        return super.hashCode();
    }
}
